npm/webpack-dev-server/package.json

```diff
@@ -18,7 +18,7 @@
     "test-unit": "mocha -r ts-node/register/transpile-only --config ./test/.mocharc.js"
   },
   "dependencies": {
-    "find-up": "6.3.0",
+    "find-up": "5.0.0",
```

Comment:
I was thinking it was because `6.3.0` is ESM only, and `webpack-dev-server` is not an ESM project - though I guess we're using `importModule` so that shouldn't matter